.github/renovate.json

```diff
@@ -0,0 +1,36 @@
+{
+  "extends": [
+    "config:base",
+    ":enablePreCommit",
+    ":preserveSemverRanges",
+    ":disableDependencyDashboard"
+  ],
+  "labels": ["dependencies"],
+  "packageRules": [
+    {
+      "matchManagers": ["github-actions"],
+      "addLabels": ["ci"]
+    },
+    {
+      "matchManagers": ["composer"],
+      "addLabels": ["php"]
+    },
+    {
+      "matchManagers": ["pip_setup", "pip_requirements"],
+      "addLabels": ["python"]
+    },
+    {
+      "matchPaths": ["website/**"],
+      "addLabels": ["javascript"],
+      "ignoreDeps": [
+        "@docusaurus/module-type-aliases",
+        "@docusaurus/preset-classic"
+      ]
```

Comment:
Isn't renovate capable of bumping multiple dependencies at once ? Just like bumping the react version ?

Reply:
It can. Will have a look at lumping these into a monorepo config

Reply:
There is already one provided :tada: